packages/atomic/src/components/atomic-result-list-placeholder/atomic-result-list-placeholder.tsx

```diff
@@ -38,18 +39,23 @@ export class AtomicResultListPlaceholder implements InitializableComponent {
   public render() {
     const placeholders = [];
     for (let i = 0; i < this.resultPerPageState.numberOfResults; i++) {
+      const opacity = `${getRandomArbitrary(0.25, 1)}`;
```

Comment:
For those magic number used to `getRandomArbitrary` (here and for all placeholders), are they from mockups ?

And why is the opacity here slightly different than for `facet-placeholder` ? (0.3, 1)

Reply:
I loosely based myself off the mockup, trial and error what looked better